jscomp/test/test_tagged_template.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+var Sql = require("sql").default;
```

Comment:
I'm going to rename this file to `tagged_template_test.js` and then have there be an `tagged_template_sql.js` file that contains an actual `sql` tagged template function so that we can test that the compile code is calling it correctly.